Source/ACE.Entity/AceObjectPropertiesSpell.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using ACE.Common;
+using MySql.Data.MySqlClient;
+namespace ACE.Entity
+{
+    [DbTable("ace_object_properties_spell")]
```

Comment:
I'm not sure I understand why we need a new table to represent spells -- an integer property of an object -- when we already have an int_properties table.  I'm not saying its wrong, I just don't immediately see why its needed